CMakeLists.txt

```diff
@@ -359,6 +359,13 @@ elseif(CELERITAS_USE_HIP)
   target_include_directories(celeritas_device_toolkit
     SYSTEM INTERFACE "${ROCM_PATH}/include"
   )
+  find_library(ROCTX_LIBRARY roctx64 HINTS ${ROCM_PATH}/lib64)
+  if(ROCTX_LIBRARY)
+    set(CELERITAS_HAVE_ROCTX ON)
+    target_link_libraries(celeritas_device_toolkit INTERFACE ${ROCTX_LIBRARY})
+  else()
+    message(STATUS "Roctx library not found. ScopedProfiling has no effect")
```

Comment:
Replace the configure-time warning (and *every reconfigure*-time warning) with the runtime warning that shows up only when the user requests profiling.